sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java

```diff
@@ -733,4 +761,45 @@ private DisplayData build() {
 
     return Item.create(key, type, value);
   }
+
+  /**
+   * Wraps exceptions passed via {@link #from(Throwable)}.
+   *
+   * @see #from(Throwable)
+   */
+  static class DisplayDataException extends Exception implements HasDisplayData {
```

Comment:
This is simply to have a stable namespace for the registered DisplayData. It is package-private and not exposed through any APIs.